### ISSUE TITLE
Issue with HTTP Request node in version 0.148.0, status code 411.

### ISSUE THREAD

**Issue by Yaccobv2** [n8n-io/n8n#2423]
Hi! I think there is an issue with the http request node in version 0.148.0. 

Every time I try to send binary data, the server responds with status code of 411.

{ "message": "Request failed with status code 411", "name": "Error", "stack": "Error: Request failed with status code 411\n at createError (/usr/local/lib/node_modules/n8n/node_modules/axios/lib/core/createError.js:16:15)\n at settle (/usr/local/lib/node_modules/n8n/node_modules/axios/lib/core/settle.js:17:12)\n at IncomingMessage.handleStreamEnd (/usr/local/lib/node_modules/n8n/node_modules/axios/lib/adapters/http.js:269:11)\n at IncomingMessage.emit (events.js:327:22)\n at endReadableNT (internal/streams/readable.js:1327:12)\n at processTicksAndRejections (internal/process/task_queues.js:80:21)" }

Configuration:
```
 {
    "parameters": {
      "requestMethod": "POST",
      "url": "=https://api.trello.com/1/cards/{{$node[\"Trello\"].json[\"id\"]}}/attachments?key=***&token=***",
      "allowUnauthorizedCerts": true,
      "jsonParameters": true,
      "options": {
        "bodyContentType": "multipart-form-data",
        "fullResponse": true
       },
      "sendBinaryData": true
     },
      "name": "TRELLO ADD ATTACHMENT",
      "type": "n8n-nodes-base.httpRequest",
      "typeVersion": 1,
      "position": [
        2160,
        270
       ],
      "continueOnFail": true
    },
```
I think there is a problem with  contentLength header property.
Hope you are able to solve this issue or guide me how should I resolve it in my workflow :) (CC @dolohow)

**Comment by janober** [comment]
Very sorry to hear that you have problems. Can you please try setting the environment variable `N8N_USE_DEPRECATED_REQUEST_LIB=true` and see if it works then.

Thanks a lot!

**Comment by Yaccobv2** [comment]
It helped, thanks a lot :)

**Comment by janober** [comment]
Thanks for confirming. We will have a look. Assigning @krynble

**Comment by krynble** [comment]
Thanks for reporting.

I wrote a fix on PR https://github.com/n8n-io/n8n/pull/2427

This should be released soon and you can then remove the flag mentioned by Jan.

I will be closing this issue as the bug was fixed.

**Comment by janober** [comment]
Got released with n8n@0.149.0
